api/src/main/resources/workbench-api.yaml

```diff
@@ -3604,6 +3604,8 @@ definitions:
         "$ref": "#/definitions/DataAccessLevel"
       archivalStatus:
         "$ref": "#/definitions/ArchivalStatus"
+      microarrayBigqueryDataset:
```

Comment:
Now that I'm looking at this, the client won't really be able to do much with it (they don't know the BQ project). The only thing the client needs to use this for is to make a determination as to which features should be shown in a given workspace.

WDYT about changing this to be a boolean `hasMicroarrayData`?

Reply:
sounds good. I'll just make the change for the API model